src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,178 @@
+package duke;
+
+import java.util.List;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * The Ui class for Duke.
+ *
+ * It consists of methods that deals with interaction with users.
+ *
+ * CS2103T IP
+ * AY22/23 Semester 1
+ * @author Tan Jia Rong
+ */
+public class Ui {
+    private static final String BOT_NAME = "Duke";
+
+    /**
+     * Prints Greetings Message.
+     */
+    public void greetings() {
```

Comment:
I believe the course's [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#naming) requires us to use verbs for methods.

> Names representing methods must be verbs and written in camelCase.
> ```getName(), computeTotalWidth()```

Perhaps we could use something like `sendGreeting()` or `greet()`?

This issue is repeated in subsequent lines of `Ui.java` as well:
 * `farewell()` -> `sendFarewell` or `bidFarewell`?
 * `helpMessage()` -> `sendHelpMessage` or `showHelpMessage`?

Reply:
Thank you I have since updated my method Names ❤️